src/algebra/group/basic.lean

```diff
@@ -337,6 +337,15 @@ calc  a / 1 = a * 1⁻¹ : div_eq_mul_inv a 1
           ... = a * 1 : congr_arg _ one_inv
           ... = a     : mul_one a
 
+@[simp, to_additive neg_sub]
+lemma inv_div' (a b : G) : (a / b)⁻¹ = b / a :=
+inv_eq_of_mul_eq_one ( by rw [div_eq_mul_inv, div_eq_mul_inv, mul_assoc, inv_mul_cancel_left,
+  mul_right_inv])
+
+@[simp, to_additive sub_add_cancel]
+lemma div_mul_cancel' (a b : G) : a / b * b = a :=
+by rw [div_eq_mul_inv, inv_mul_cancel_right a b]
+
 end group
 
 section add_group
```

Comment:
Note the comment below this line. If you're going to additivize some of these, could you stretch to doing all of them?

Reply:
Possibly - in fact I think I've got a branch on my computer that already does that from https://github.com/leanprover-community/mathlib/pull/8673#discussion_r693396835

Am I handling the name collisions with `group_with_zero` correctly?

Reply:
I found the branch where I started work on this some time ago, and have pushed it: https://github.com/leanprover-community/mathlib/tree/sub-to-div

I can't quite remember where I got to, but I expect it needs a bit more work.

Reply:
Here you are: https://github.com/leanprover-community/mathlib/pull/10532

Reply:
And here's the rest of it https://github.com/leanprover-community/mathlib/pull/10565